### ISSUE TITLE
When using as ovirt nfs storage domain, creating vm from template fails

### ISSUE THREAD

**Issue by nirs** [nirs/slowfs#1]
4004714a-7c9a-4103-b973-65dfcbc59057::ERROR::2015-10-21 16:06:06,848::task::866::Storage.TaskManager.Task::(_setError) Task=`4004714a-7c9a-4103-b973-65dfcbc59057`::Unexpected error
Traceback (most recent call last):
  File "/usr/share/vdsm/storage/task.py", line 873, in _run
    return fn(_args, *_kargs)
  File "/usr/share/vdsm/storage/task.py", line 334, in run
    return self.cmd(_self.argslist, *_self.argsdict)
  File "/usr/share/vdsm/storage/securable.py", line 77, in wrapper
    return method(self, _args, *_kwargs)
  File "/usr/share/vdsm/storage/sp.py", line 1821, in createVolume
    desc=desc, srcImgUUID=srcImgUUID, srcVolUUID=srcVolUUID)
  File "/usr/share/vdsm/storage/sd.py", line 430, in createVolume
    preallocate, diskType, volUUID, desc, srcImgUUID, srcVolUUID)
  File "/usr/share/vdsm/storage/volume.py", line 412, in create
    (srcVolUUID, volUUID, e))
VolumeCannotGetParent: Cannot get parent volume: ("Couldn't get parent 2cff7724-9874-41ee-9025-3226a2bc40e8 for volume 39a87ea6-8d8f-4138-84a9-9c2c222ba7de: Cannot share volume: u'src=/rhev/data-center/00000002-00
02-0002-0002-000000000152/f1260a63-d301-465d-a41e-a1986c6f702a/images/ccc99a6d-1818-4d6d-b27d-7cb0ebe7ab76/2cff7724-9874-41ee-9025-3226a2bc40e8, dst=/rhev/data-center/00000002-0002-0002-0002-000000000152/f1260a63-
d301-465d-a41e-a1986c6f702a/images/b8ce7687-a665-45e9-8523-b8df118b7c03/2cff7724-9874-41ee-9025-3226a2bc40e8: [Errno 2] No such file or directory'",)

**Comment by nirs** [comment]
May be relaed:

``` python
177     def link(self, target, name):                                                                                                                                           
178         return os.link(self._full_path(target), self._full_path(name))
```

The code may assume relative paths, and this converts paths to absolute - is this really needed?

**Comment by nirs** [comment]
Fixed in c45e727
